### ISSUE TITLE
Hex color specifications are not recognized

### ISSUE THREAD

**Issue by SlimeDog** [AlonsoAliaga/BetterBees#10]
**Describe the bug**
Hex color specifications are not recognized on `reload`. Actual color does not matter -- any hex value causes the following error, including the unedited default color "ebdc13" and "".
```
> betterbees reload
[07:09:50 INFO]: [BetterBees] Color in 'Options.Bees.Path-to-hive.Color' path is not valid. Using default color..
[07:09:50 INFO]: [BetterBees] Color in 'Options.Bees.Path-to-flower.Color' path is not valid. Using default color..
```

config.yml
```
...
Options:
  Bees:
    Path-to-hive:
      #If enabled, it will show a line between bee and his hive location if available.
      Enabled: true
      #Color for particles to hive. Color must be in Hex format. Do NOT include the #
      #You can use this https://www.google.com/search?q=color+pick
      Color: "ebdc13"
      #Time in seconds the path to hive will be displayed. Default 2
      Time: 2
    Path-to-flower:
      #If enabled, it will show a line between bee and his flower location if available.
      Enabled: true
      #Color for particles to flower. Color must be in Hex format. Do NOT include the #
      #You can use this https://www.google.com/search?q=color+pick
      Color: "eb2113"
      #Time in seconds the path to flower will be displayed. Default 2
      Time: 2
...
```

**To Reproduce**
1. Execute `betterbees reload` in-game or at console
2. Read console/log error (no error shown in-game)

**Expected behavior**
Should be able to specific path colors.

**Important (please complete the following information):**
 - Java version: 17.0.1
 - Plugin version: 0.8-BETA
 - Server version: Spigot 1.18.1-3377 or Paper 1.18.1-101

**Comment by AlonsoAliaga** [comment]
Error has been found and fixed on 0.9-BETA (Pending to release) 😁

**Comment by AlonsoAliaga** [comment]
✅ Version 0.9-BETA available now on [SpigotMC](https://alonsoaliaga.com/BetterBees)

Thanks for reporting!
